packages/@aws-cdk/aws-elasticloadbalancingv2/test/alb/target-group.test.ts

```diff
@@ -19,6 +19,23 @@ describe('tests', () => {
     }).toThrow(/'vpc' is required for a non-Lambda TargetGroup/);
   });
 
+  test('Lambda target should not have stickiness.enabled set', () => {
+    const app = new cdk.App();
+    const stack = new cdk.Stack(app, 'Stack');
+    new elbv2.ApplicationTargetGroup(stack, 'TG', {
+      targetType: elbv2.TargetType.LAMBDA,
+    });
+
+    expect(stack).not.toHaveResource('AWS::ElasticLoadBalancingV2::TargetGroup', {
```

Comment:
You'll want `toHaveResourceLike` here. Otherwise, this just checks to make sure there's not a `TargetGroup` with _only_ the attributes you specify

Reply:
@peterwoodworth Hmm, this doesn't seem to work anymore; maybe related to https://github.com/aws/aws-cdk/pull/17344? What can I use to test that a certain field does not exist in the resource?

```
    Not None of 1 resources matches resource 'AWS::ElasticLoadBalancingV2::TargetGroup' with {
      "$deepObjectLike": {
        "TargetGroupAttributes": [
          {
            "Key": "stickiness.enabled"
          }
        ]
      }
    }.
    - Field TargetGroupAttributes missing in:
        {
          "Type": "AWS::ElasticLoadBalancingV2::TargetGroup",
          "Properties": {
            "TargetType": "lambda"
          }
        }
```

Reply:
Never mind, it was just that I forgot to run `yarn bulid` before running `yarn test` 🤦‍♂️